packages/components/psammead-image-placeholder/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@bbc/psammead-image-placeholder",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "description": "Provides a 'BBC' image placeholder",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image-placeholder"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-image-placeholder/README.md",
+  "dependencies": {
+    "@bbc/psammead-assets": "^0.1.2",
+    "@bbc/psammead-styles": "^0.1.3",
+    "react": "^16.6.3",
```

Comment:
React is a dev dependancy, not a normal one

Reply:
@dr3 your comment is addressed.